analysis_options.yaml

```diff
@@ -0,0 +1,45 @@
+include: package:lint/analysis_options.yaml
+
+analyzer:
+  strong-mode:
+    implicit-casts: false
+    implicit-dynamic: false
+  errors:
+    prefer_const_constructors: warning
+  exclude:
+    - "**/*.g.dart"
+    - "**/*.freezed.dart"
+    - "**/*.config.dart"
+    - "lib/gen/**"
+    - "test/**"
```

Comment:
should we really skip tests? 
non blocker tho!

Reply:
I think we shouldn't enforce the same standards on tests. Since the tests are localized and often small.

Reply:
tests are first class citizens

Reply:
okay will add lints to them